### PR TITLE
feat: add dotenv for server integration

### PR DESCRIPTION
**Pull request by BryanPan342**
- not gitignoring the `.env` file for now so it appears on the server.. next commit we will gitignore it
- add cloudflare to go to ip address

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const PLAYER = document.querySelector('#player');
 /* eslint-enable no-undef, no-unused-vars */
 
 function setupNetworking() {
-  const serverUrl = process.env.NODE_ENV === 'production' ? 'TODO' : 'http://localhost:3000';
+  const serverUrl = process.env.NODE_ENV === 'production' ? 'http://yummy-server.creativelabsucla.com/' : 'http://localhost:3000';
```
Comment (arjunr00) [inline client/src/index.js:8]:
```suggestion
  const serverUrl = process.env.NODE_ENV === 'production' ? 'http://chef.creativelabsucla.com/' : 'http://localhost:3000';
```

Change to this?

**Review by arjunr00 — CHANGES_REQUESTED** [review]
Change

File: client/src/index.js (hunk 1 above)
Comment (BryanPan342) [reply client/src/index.js:8]: should we do this in a chore? when we change everything in the repo to chef?

Comment (arjunr00) [reply client/src/index.js:8]: yeah sounds good, its not a big deal anyway

**Review by arjunr00 — APPROVED** [review]

**Pull request merged**